websites/L/LoL Espor/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "mertkaraca.mp4",
+    "id": "749681979741569052"
+  },
+  "service": "LoL Esports",

```

Comment:
update the folder name, and the name in discord, and upload new proof of it working then

Reply:
done